fern/docs.yml

```diff
@@ -201,20 +201,20 @@ navigation:
         snippets:
           python: fern-api
           typescript: "@fern-api/node-sdk"
-  - tab: fern-docs
+  - tab: docs
     layout:
       - page: Overview
         path: ./docs/pages/fern-docs/docs-overview.mdx
-      - section: Fern Docs Quickstarts
+      - section: Quickstarts
         slug: quickstarts
         contents:
-          - page: Fern Docs Quickstart with OpenAPI
+          - page: Use OpenAPI
             slug: openapi
             path: ./docs/pages/fern-docs/docs-quickstarts/docs-quickstart-openapi.mdx
-          - page: Fern Docs Quickstart with Fern Def
+          - page: Use a Fern Definition
             slug: ferndef
             path: ./docs/pages/fern-docs/docs-quickstarts/docs-quickstart-ferndef.mdx
```

Comment:
needs to eventually be a single quickstart that leverages two cards and a stepper (prisma does this) https://www.prisma.io/docs/getting-started/quickstart
![Uploading Screenshot 2024-03-20 at 7.39.22 AM.png…]()